ServerWhitelist.yml

```diff
@@ -1947,19 +1947,15 @@ AllowedPrefixes:
     - https://cdn.discordapp.com/attachments/1063514401375780874/1198357469903204383/SexLab_Framework_PPLUS_-_V2.3.2.1_-_1.5.97.zip?ex=65da4bdd&is=65c7d6dd&hm=3121fe0ab2594a7a661e07a0d19c0dafe7f72813946e2be02479e04977e51a9b& # Sexlab + 2.3.2.1 / SE 1.5.9.7 ( Inline Permission from Scrab )
     - http://www.mediafire.com/file/o5yacfkx21p3b8i/Sexlab_DLL_16640_Replacer.rar # Sexlab Framework 16640 Working DLL ( Inline Permission from Scrab )
     - https://cdn.discordapp.com/attachments/1177560192192352336/1178355556113731604/SexLab_P_Quick_Scene_Start_Spell_-_4.0.0.zip?ex=65db5ba1&is=65c8e6a1&hm=0afebe8fd8aedc5ce84171cc1b0d649fd62bbe888f4d75c89f95fcdb6c34ab4c& # Sexlab + Quick Spell Inline
-    - https://mega.nz/file/UlV1XAxJ#Fq-VsCAtNEF2kaaqeCepvKBW8AhDfbeyb5-j1RU8mWo # PAH - And You get a Slave v0.60a - Voicepack SE.7z
-    - https://mega.nz/file/p88gVISL#29j2CjGATvyJ-5Oh1lC-Kql-43tN2O7sFmTlbTTS-Qs # PAH - Home Sweet Home v1.27 - Voicepack SE.7z
-    - https://mega.nz/file/A1k2nD4D#L0wIZJ8h_yNDCJVN3XYTRoZZdD-9IURkMixEP0pwmCE # Paradise Halls v8.1.6 - Voicepack v1.0 - SE.7z
-    - https://mega.nz/file/QtkSFZYZ#m6VeiRbzW4ee8uCm1I_KbiZM7tspX_p6j2POwiYWWy4 # SexLab TDF Prostitution and Pimping 2023 - Voicepack.7z
     - https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/releases # Pandora Behaviour Engine v1.1.6-alpha
     - https://www.patreon.com/posts/edge-ui-0-55-98401985 # Edge UI 0.55 Public Release
     - https://drive.google.com/file/d/1PtavIFaxtEX8Jb_No6FczweWuYhyCMxy/view # [SE] BDOR Arditer.7z
-	- https://drive.google.com/file/d/1G5yiU0xbXh_y9XsPDQ6z5Gg6razeb-Nk/view # [SE] DM BDOR Corbicanus by Team TAL, em486.zip
-	- https://drive.google.com/file/d/1johSpGtPpts5FYRbjTZOywZNlBBmt2fE/view # [SE] BDO Oblivion.7z
-	- https://drive.google.com/file/d/1ahBFM-TLv-sy-rhwlrBChVGatNf1FRpB/view # [SE] BDOR Mountain Spirit.7z
-	- https://drive.google.com/file/d/1RjySJfGmKBOFCp-5eFXZvwXy1phhTwhT/view # [SE] BDOR La Orzeca by Team TAL(converted by em486).7z
-	- https://drive.google.com/file/d/1kWnDqqin2KZBIOYGxeBkS0LrmJ6DGKn8/view # [Christine] Dragon Marauder.7z
-	- https://mega.nz/file/9moCWSDB#PWuH6fkHj_JHquUBucIt6AZXOcWgcnMofarrq4mPJWY # SexyBono Eyes by immyneedscake
-	- https://mega.nz/file/PaRXiQBK#PZyU2kfU2XEVIiuT1B1C82MCb8sKv5Ozg8QY9ls1dsY # Morningstar Wing SMP by immyneedscake
-	- https://www.mediafire.com/file/9e000wxv51i4afw/iAm_fixes_Vominheim.rar/file # iAm_fixes_Vominheim ( Permission to uplaod )
+    - https://drive.google.com/file/d/1G5yiU0xbXh_y9XsPDQ6z5Gg6razeb-Nk/view # [SE] DM BDOR Corbicanus by Team TAL, em486.zip
+    - https://drive.google.com/file/d/1johSpGtPpts5FYRbjTZOywZNlBBmt2fE/view # [SE] BDO Oblivion.7z
+    - https://drive.google.com/file/d/1ahBFM-TLv-sy-rhwlrBChVGatNf1FRpB/view # [SE] BDOR Mountain Spirit.7z
+    - https://drive.google.com/file/d/1RjySJfGmKBOFCp-5eFXZvwXy1phhTwhT/view # [SE] BDOR La Orzeca by Team TAL(converted by em486).7z
```

Comment:
https://arca.live/b/tullius/9719430?category=%EC%9D%98%EC%83%81&target=all&keyword=BDO&p=4
Armor Mod Source

Reply:
I don't trust this source again again again.

Reply:
Will remove remove.